cloud/azure/storage-account-capacity/detectors-azure-storage.tf

```diff
@@ -0,0 +1,30 @@
+resource "signalfx_detector" "storage_capacity" {
+  name = format("%s %s", local.detector_name_prefix, "Azure Storage Account used capacity")
+
+  program_text = <<-EOF
+        base_filter = filter('resource_type', 'Microsoft.Storage/storageAccounts') and filter('primary_aggregation_type', 'true')
+        signal = data('UsedCapacity', filter=base_filter and ${module.filter-tags.filter_custom})${var.storage_capacity_aggregation_function}.publish('signal')
```

Comment:
why not using https://developers.signalfx.com/signalflow_analytics/methods/scale_stream_method.html to convert bytes into GB ?